playground/gatsby-config.js

```diff
@@ -38,18 +40,23 @@ module.exports = {
     {
       resolve: require.resolve(`../`),
       options: {
-        host: 'http://127.0.0.1:7700',
-        apiKey: 'masterKey',
-        skipIndexing: true,
+        host:
+          process.env.GATSBY_MEILI_SERVER_ADDRESS || 'http://localhost:7700',
```

Comment:
```suggestion
          process.env.GATSBY_MEILI_SERVER_ADDRESS || 'http://localhost:7700',
```

I'm wondering if these environment variables names should be the same as the ones used by MeiliSearch: https://docs.meilisearch.com/reference/features/configuration.html#passing-arguments-via-the-command-line

Same question for all env variables used in this project

Reply:
You're right, it's done ❤️